packages/components/src/styles/components/app-header.scss

```diff
@@ -107,23 +107,27 @@
     gap: inherit;
   }
 
-  // Apply dark theme to child interactive components
+  // Dropdown & Button color theming overrides
   .hds-button,
   .hds-dropdown-toggle-button,
   .hds-dropdown-toggle-icon {
-    @include hds-interactive-dark-theme();
-
-    // disabled state:
-    &:disabled,
-    &[disabled],
-    &.mock-disabled,
-    &:disabled:focus,
-    &[disabled]:focus,
-    &.mock-disabled:focus,
-    &:disabled:hover,
-    &[disabled]:hover,
-    &.mock-disabled:hover {
-      @include hds-interactive-dark-theme-state-disabled();
+    // Apply dark theme to child interactive components not within a nested dropdown
+    // NOTE: Testing overrides for ember-basic-dropdown as well although likely not something we want to do
+    &:not(.hds-dropdown * *, .ember-basic-dropdown-trigger * *, .ember-basic-dropdown-content * *) {
```

Comment:
I included selectors to target ember-basic-dropdown since the Vault team chose to create their own custom Dropdown-like component using it instead of using the HDS Dropdown. However, I'm not sure if it's really advisable to include class names from third party components like this. (_**Please let me know your opinions.**_)

**Note:** These complex scoping styles aren't ideal but seem to be a workable solution for now. Ideally, once we begin to implement theming, we can refactor to remove the need for them.

Reply:
Well, we do use these third-party classes in the SuperSelect, so I wouldn't worry that much.

Reply:
Well, we do use these third-party classes in the SuperSelect, so I wouldn't worry that much.